packages/neuron-ui/src/locales/en.json

```diff
@@ -1216,12 +1219,11 @@
     },
     "main": {
       "external-node-detected-dialog": {
-        "title": "External node detected",
-        "body-tips-without-network": "\"Internal Node\" is reserved for the built-in CKB node, please add a new network option for the external node.",
-        "body-tips-with-network": "\"Internal Node\" is reserved for the built-in CKB node, please select from the following network options or add a new one for the external node.",
+        "title": "Detected external node",
+        "body-tips-without-network": "You have selected the internal node but started the external node, if you want to continue to use an external node, please add a new network.",
```

Comment:
The message isn't clear. Does it indicate a conflict between the external node and the Neuron built-in node due to the same port?

Reply:
cc @Danie0918

Reply:
> The message isn't clear. Does it indicate a conflict between the external node and the Neuron built-in node due to the same port?

Regarding the ports being the same, the built-in nodes automatically adjust the ports, right? @yanguoyu

Reply:
Right